### ISSUE TITLE
"Using eval in Chrome extensions" page is very out of date

### ISSUE THREAD

**Issue by fixermark** [GoogleChrome/developer.chrome.com#1874]
⚠️ This is for issues with the https://developer.chrome.com site, not Chromium itself.
If you want to file a bug with Chromium (the open-source project behind Google Chrome and other browsers), please use https://crbug.com.

**Describe the bug**
"Using eval in Chrome extensions" is very out of date

**To Reproduce**
1. Go to https://developer.chrome.com/docs/extensions/mv3/sandboxingEval/

**Expected behavior**
Documentation should describe the steps needed to use `eval` in Chrome extensions under Manifest V3 

**Observed behavior**
Documentation is still Manifest V2. A large banner at the top declares

```
This page was migrated directly from the MV2 documentation set. It has not yet been validated for compliance with Manifest V3.
```

This is disquieting given that MV2 is on-track to be unusable for new extensions in a year.

**Screenshots**

**Desktop (please complete the following information):**
 - OS: Windows
 - Browser Chrome
 - Version 96.0.4664.110

**Comment by jpmedley** [comment]
@AmySteam Do your updates cover this?

**Comment by AmySteam** [comment]
Fixed in #6546
